src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Validators/UserInfoValidatorTests.cs

```diff
@@ -28,7 +28,7 @@ public void Validate_UserName_ShouldBeValidated(string value, bool expectedValid
 
         [TestCase("CorrectEmail@here.com", true)]
         [TestCase("IncorrectEmail", false)]
-        [TestCase("IncorrectEmail@nowhere", false)]
+        [TestCase("IncorrectEmail@nowhere", true)]
```

Comment:
should we just be changing the expected result of this test?

Reply:
FluentValidation have changed their email validator to be less strict. Technically the test I changed is a valid email address so should pass.